.circleci/config.yml

```diff
@@ -98,6 +98,42 @@ jobs:
               echo "Skip the Artman Docker image publishing step."
             fi
 
+  release:
+    working_directory: /usr/src/artman
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - *checkout_artman
+      - setup_remote_docker
+      - *restore_artman_cache
+      - *load_build_and_save_artman_image
+      - *save_artman_cache
+      - run:
+          name: Decrypt PyPI configuration file
+          command: |
+            openssl aes-256-cbc -d -in .circleci/.pypirc.enc -out ~/.pypirc -k "${PYPIRC_ENCRYPTION_KEY}"
+      - run:
+          name: Push Artman Docker tagged image
+          command: |
+            if [ "${DOCKER_EMAIL}" == 'googleapis-publisher@google.com' ]; then
+              # extract version number from setup.py
+              ARTMAN_VERSION=`awk -F"'" '/version=/ { print $2; }' setup.py`
```

Comment:
Do we perhaps want to add a note into setup.py that this version line is scraped for this purpose?

Reply:
I actually have an idea for some refactoring of `setup.py` and `Dockerfile` related to versions and commit hashes, will update this PR with that.